rdr_service/genomic/genomic_job_components.py

```diff
@@ -1101,6 +1101,31 @@ def _process_gc_metrics_data_for_insert(self, rows):
 
         return GenomicSubProcessResult.SUCCESS
 
+    def copy_member_for_replating(self, member, genome_type=None, set_id=None):
+        """
+        Inserts a new member record for replating.
+        :param member: GenomicSetMember
+        :return:
+        """
+        new_member = GenomicSetMember(
+            biobankId=member.biobankId,
+            genomicSetId=set_id if set_id else member.genomicSetId,
+            participantId=member.participantId,
+            nyFlag=member.nyFlag,
+            sexAtBirth=member.sexAtBirth,
+            validationStatus=member.validationStatus,
+            validationFlags=member.validationFlags,
+            ai_an=member.ai_an,
+            genomeType=genome_type if genome_type else member.genomeType,
+            collectionTubeId=f'replated_{member.id}',
+            genomicWorkflowState=GenomicWorkflowState.EXTRACT_REQUESTED,
+            created=clock.CLOCK.now(),
+            modified=clock.CLOCK.now(),
```

Comment:
Probably can remove this in lieu of the listener on the model.

Reply:
Removed.